Fw/SerializableFile/CMakeLists.txt

```diff
@@ -12,13 +12,10 @@ set(SOURCE_FILES
   "${CMAKE_CURRENT_LIST_DIR}/SerializableFile.cpp"
 )
 set(MOD_DEPS
-  Fw/Cfg
-  Fw/Types
+    Fw/Cfg
+    Fw/Types
+    Os
 )
-# For shared libraries, Os must exist in the module list
```

Comment:
This looks fine, but what changed to cause the deletion?

Reply:
Os needed to be track as an actual link dependency.  Before it worked as a quirk of other ordering, but ordering changes revealed that it needed to be explicit....not just used for shared libs.

Reply:
Thanks!